src/clr-core/styles/tokens/tokens.yml

```diff
@@ -179,6 +179,66 @@ props:
     type: 'number'
     category: 'typography'
 
+  # global font sizes
```

Comment:
Added these to match the weight tokens. These were useful for forms where there are labels and elements that dont directly match the content attributes applications would use.

Reply:
Could we mark these as something other than `typography`? It sounds like we want to use them internally. But they are presented as an alternative to the typography tokens that we want applications to use.

Also, I had/have a need for `11px` in buttons. Do we go there? Or should I just calc() it? Where should we draw the line on how many of these we offer?

Reply:
I can make them all private and internal. Thats what they were intended for. I'll add 11px and 13px as well since those are used in a lot of places. If we mark these internal then we can change them at anytime.